components/support/nimbus-fml/fixtures/fe/fenix.yaml

```diff
@@ -1,4 +1,12 @@
 ---
+about:
+  description: This file codes for all features in the Fenix browser
+  kotlin:
+    class: .nimbus.FxNimbus
+    package: com.example.app
+  swift:
+    class: MyNimbus
+    module: Client
```

Comment:
I realize that this is just a test fixture, but it strikes me as a bit strange to see "swift" here for Fenix